submodules/runtimes/rust/dafny_impl/Cargo.toml

```diff
@@ -6,7 +6,8 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+chrono = "0.4.38"
```

Comment:
The `dafny_impl` folder will eventually be auto generated, including, this file. Is there any way to specify this dependency in another crate that the extern would somehow automatically refer to?
Else we can always change the build script to include this dependency, no worries

Reply:
Implementing externs will always involve hand-editing some Cargo.toml file.
I'll try to figure out a way for it not to be this one.